### PR TITLE
Changed Twitter to X in personal portfolio

### PR DESCRIPTION
**Pull request by AlzyWelzy**
I have changed Twitter to X in the personal portfolio section.

### CODE REVIEW

**Comment by CouchofTomato** [comment]
There are merge conflicts with this PR.

**Comment by wise-king-sullyman** [comment]
@AlzyWelzy are you planning to resolve the merge conflicts in this PR?

File: advanced_html_css/responsive_design/project_personal_portfolio.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Also, since this is your _personal_ portfolio feel free to tweak the design howe
 
 1. The portraits we've used in the design files are stock photos downloaded from [pexels.com](https://www.pexels.com/). If you don't have a picture of yourself handy, feel free to go grab a placeholder for now.
 2. Select your fonts! We're using `Playfair Display` and `Roboto` in the design, both available with Google fonts.
-3. In the design we have icon-links for GitHub, LinkedIn and Twitter. Obviously feel free to add whatever links you want to your own site. We got those icons from [https://devicon.dev/](https://devicon.dev/).
+3. In the design we have icon-links for GitHub, LinkedIn and X. Obviously feel free to add whatever links you want to your own site. We got those icons from [https://devicon.dev/](https://devicon.dev/).
```
Comment (wise-king-sullyman) [inline advanced_html_css/responsive_design/project_personal_portfolio.md:23]: There are merge conflicts which need to be resolved.

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]

**Comment by wise-king-sullyman** [comment]
I'm a bit confused, it looks like the merge conflict still needs to be resolved and you've added conduct code and contributing files?

**Comment by wise-king-sullyman** [comment]
Closing due to inactivity.

**Pull request closed without merging**